### ISSUE TITLE
Why i facing this issue with pc or wide screens ?  example attached 

### ISSUE THREAD

**Issue by Nazeh-Taha** [vre2h/use-react-screenshot#56]
with regular laptop screen :
![Annotation 2023-03-07 094557](https://user-images.githubusercontent.com/54739625/223358449-6d086567-a29f-439b-a1a7-77534a8945e5.png)

with 17-inch screen :
![1](https://user-images.githubusercontent.com/54739625/223358646-740a7b37-7f01-4c5b-9558-43898c71b489.png)

**Comment by ishwark7** [comment]
Even I am facing similar issue, can we have any update on this?

**Comment by Nazeh-Taha** [comment]
Yes, I am using material ui the issue happened because using the <Paper /> component to wrap the content

**Comment by benhammondmusic** [comment]
> Yes, I am using material ui the issue happened because using the `<Paper />` component to wrap the content

We are having this issue as well; any idea what the source of conflict is between material ui and this screenshot package?
